TELCODOCS-386: Send SRO metrics as telemetry data (#41158)
openshift/openshift-docs: modules/psap-special-resource-operator-using-configmaps.adoc

//
// * hardware_enablement/psap-special-resource-operator.adoc

:_content-type: PROCEDURE
[id="deploy-simple-kmod-using-configmap-chart"]
= Building and running the simple-kmod SpecialResource by using a config map

> In this example, the simple-kmod kernel module is used to show how the Special Resource Operator (SRO) can manage a driver container which is defined in Helm chart templates stored in a config map.

Reviewer: ```suggestion
In this example, the simple-kmod kernel module shows how the Special Resource Operator (SRO) manages a driver container. The driver container is defined in the Helm chart templates that are stored in a config map.
```

This sentence is going to be hard to follow for someone who is not a native speaker. Suggestion above to break it up into two sentences for simpler reading and to apply minimalism.

Author: Agree. I've not tried to reword/rework these sentences, just made some tweaks I needed for my changes to work.  I'll apply this and the rest of the rewording suggestions, they will make the assembly better. 

Thanks!

----
$ oc create -f simple-kmod-configmap.yaml
----

[NOTE]
====

> If you want to remove the simple-kmod kernel module from the node, delete the simple-kmod `SpecialResource` API object using the `oc delete` command. The kernel module is unloaded when the driver container pod is deleted.

Reviewer: ```suggestion
To remove the simple-kmod kernel module from the node, delete the simple-kmod `SpecialResource` API object using the `oc delete` command. The kernel module is unloaded when the driver container pod is deleted.
```
Applying minimalism principles.

simple_kmod            16384  0
----

[TIP]

> ===

Author: tips seem to be missing one =
